explainaboard/processors/processor.py

```diff
@@ -139,36 +138,25 @@ def _gen_external_stats(
                     sys_info.dataset_name, sub_dataset
                 )
             if statistics is None:
+                dataset: FileLoaderReturn | None = None
                 try:
-                    dataset = load_dataset(sys_info.dataset_name, sub_dataset)
+                    loader = get_loader_class(self.task_type()).from_datalab(
+                        DatalabLoaderOption(
+                            sys_info.dataset_name, sub_dataset, split=split_name
+                        ),
+                        output_data=None,
+                    )
+                    dataset = loader.load()
                 except Exception:
-                    dataset = None
+                    pass
```

Comment:
It also looks this block captures any unexpected exceptions. Please narrow the exception type to focus on only expected exceptions, or add comments why this exception can be ignored safely.